class Animal {

    void makeSound() {
        System.out.println("Animal makes sound");
    }
}

class Dog extends Animal {

    void makeSound() {
        System.out.println("Bark");
    }
}

public class InheritanceDemo {

    public static void main(String[] args) {

        Animal a = new Animal();
        Dog d = new Dog();

        a.makeSound();
        d.makeSound();
    }
}

/*
Animal makes sound
Bark
*/